cli/README.md

```diff
@@ -0,0 +1,176 @@
+<div align="center">
```

Comment:
@capjamesg do you think the header in this file is necessary since this is a sub-README? Or do you think it's just repeated info?